components/Navigation/Navigation.js

```diff
@@ -1,22 +1,126 @@
 import Link from "next/link";
+import { useRouter } from "next/router";
 import styled from "styled-components";
+import Image from "next/image";
+import BookmarkIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
 
 export default function Navigation() {
+  const router = useRouter();
+  const isOnBookmarksPage = router.pathname === "/bookmarks";
```

Comment:
for the moment this is fine, but you could generalize this for any number of pages.
Also, since your special nav item is the home page you should check for that instead.